src/scene/vcFlythrough.cpp

```diff
@@ -254,7 +254,21 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
 
   if (m_state != vcFTS_Exporting)
   {
-    if (ImGui::Button(vcString::Get("flythroughExport")))
+    if (ImGui::BeginCombo(vcString::Get("flythroughFPS"), udTempStr_CommaInt(pProgramState->supportedVideoFPSs[pProgramState->videoFPSIndex])))
```

Comment:
You should just use `udTempStr` instead as `udTempStr_CommaInt` does more work for no value - it is incredibly unlikely that a video will have more than 1000 frames per second.